package.json

```diff
@@ -55,6 +55,7 @@
     "generate:proxy:mitvoucher": "rimraf generated/mitvoucher && gen-api-models --api-spec api_mit_voucher.yaml --out-dir generated/mitvoucher",
     "generate:proxy:zendesk-models": "rimraf generated/zendesk && gen-api-models --api-spec api_zendesk.yaml --out-dir generated/zendesk",
     "generate:proxy:third-party-service": "rimraf generated/third-party-service && gen-api-models --api-spec openapi/consumed/api-third-party.yaml --out-dir generated/third-party-service --request-types --response-decoders --client",
+    "generate:proxy:remote_content_service": "rimraf generated/remote_content_service && gen-api-models --api-spec openapi/consumed/api_remote_content.yaml --out-dir generated/remote_content_service --request-types --response-decoders --client",
```

Comment:
maybe better `remote-content-service` like others

Reply:
Fixed in [2ab2bea](https://github.com/pagopa/io-backend/pull/1077/commits/2ab2beab5d43f29d5394b91bc72de2812f8804c1)